src/runtime_src/core/include/xrt/xrt_aie.h

```diff
@@ -58,6 +58,14 @@ class device : public xrt::device
    */
   enum class buffer_state { idle, running, completed };
 
+  /**
+   * @enum thermal - aie thermal type
+   * 
+   * @var temperature
+   *   Thermal zone temperature
+   */
+  enum class thermal { temperature };
```

Comment:
I would probably use `thermal_type` here, or _category, or something better.   I have a comment about `arg` later which you could then name `thermal`.